projects/payment-lib/src/lib/components/fee-summary/fee-summary.component.ts

```diff
@@ -64,13 +64,8 @@ export class FeeSummaryComponent implements OnInit {
     this.selectedOption = this.paymentLibComponent.SELECTED_OPTION.toLocaleLowerCase();
     this.isStrategicFixEnable = this.paymentLibComponent.ISSFENABLE;
 
-    if ((!this.isOldPcipalOff && this.isNewPcipalOff)) {
-      this.platForm = '8x8';
-    } else if ((this.isOldPcipalOff && !this.isNewPcipalOff)) {
-      this.platForm = 'Antenna';
```

Comment:
Here, you should retain this logic for Antenna. So what will be the value for platform?

Reply:
We removed condition and assign the value 'Antenna' to platform.